rest_framework_dataclasses/serializers.py

```diff
@@ -149,7 +150,7 @@ def dataclass_definition(self) -> DataclassDefinition[T]:
             dataclass_type = meta.dataclass
 
         # Make sure we're dealing with an actual dataclass.
-        if not dataclasses.is_dataclass(dataclass_type):
+        if not dataclasses.is_dataclass(cast(Type[T], dataclass_type)):
```

Comment:
I had mypy error here and wasn't sure why so meanwhile I casted it.
https://github.com/oxan/djangorestframework-dataclasses/actions/runs/4707423442/jobs/8349271383

Reply:
I think this is a result of mypy updating and changing their rules, and unrelated to this PR... I'll look into it.